lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -260,12 +261,13 @@ def declare_suspicious_replicas_bad(once=False, younger_than=3, nattempts=10, vo
                         for replica_value in recoverable_replicas[vo][rse_key].values():
                             surls_list.append(replica_value['surl'])
 
-                        add_bad_pfns(pfns=surls_list, account=InternalAccount('root', vo=vo), state='TEMPORARY_UNAVAILABLE', expires_at=datetime.utcnow() + timedelta(days=3))
+                        if not passive_mode:
+                            add_bad_pfns(pfns=surls_list, account=InternalAccount('root', vo=vo), state='TEMPORARY_UNAVAILABLE', expires_at=datetime.utcnow() + timedelta(days=3))
 
                         logger(logging.INFO, "%s is problematic (more than %s suspicious replicas). Send a Jira ticket for the RSE (to be implemented).", rse_key, limit_suspicious_files_on_rse)
                         logger(logging.INFO, "The following files on %s have been marked as TEMPORARILY UNAVAILABLE:", rse_key)
```

Comment:
Do we still need this line, the `TEMPORARILY UNAVAILABLE` value is now included in the other log message.

Reply:
This line assigns the replicas the "temporarily unavailable" state, it doesn't add them to the log file.

Reply:
Yes, this line is where the files are declared temporarily unavailable. It doesn't include anything in the log messages.